apps/web-giddh/src/app/daybook/daybook.component.ts

```diff
@@ -146,6 +146,10 @@ export class DaybookComponent implements OnInit, OnDestroy {
         let hasParticularSelected = this.lc.blankLedger.transactions?.filter(txn => txn?.particular);
         return (hasParticularSelected?.length) ? true : false;
     }
+    /** This will use for export as file type */
+    public fileTypeExpand: string = 'CSV';
```

Comment:
check this code

Reply:
this variable was not in use so I removed variable